netbox/extras/tables/tables.py

```diff
@@ -550,12 +550,17 @@ class ConfigTemplateTable(NetBoxTable):
         url_params={'config_template_id': 'pk'},
         verbose_name=_('Virtual Machines')
     )
+    as_attachment = columns.BooleanColumn(
+        verbose_name=_('As Attachment'),
+        false_mark=None
+    )
 
     class Meta(NetBoxTable.Meta):
         model = ConfigTemplate
         fields = (
             'pk', 'id', 'name', 'description', 'data_source', 'data_file', 'data_synced', 'role_count',
-            'platform_count', 'device_count', 'vm_count', 'created', 'last_updated', 'tags',
+            'platform_count', 'device_count', 'vm_count', 'created', 'last_updated', 'tags', 'as_attachment',
+            'mime_type', 'file_name', 'file_extension',
```

Comment:
Try to order primary model fields ahead of annotations & generic fields for readability.
```suggestion
            'as_attachment', 'mime_type', 'file_name', 'file_extension','platform_count', 'device_count', 'vm_count',
            'created', 'last_updated', 'tags', 
```